packages/api/index.js

```diff
@@ -1,5 +1,17 @@
-
 const express = require('express');
+require('dotenv').config()
+const sequelize =  require( './helper/sequelize.js');
+
+async function testSequelizeConnection(){
```

Comment:
I wonder if it would make sense to add this code to the helper or the models index instead, since we'll likely want to import that `db` object instead of the raw sequelize instance here?